src/api/entities/SecurityToken/CorporateActions/index.ts

```diff
@@ -93,18 +90,12 @@ export class CorporateActions extends Namespace<SecurityToken> {
    *
    * @note this action will leave the Security Token owner as the Corporate Actions Agent
    *
-   * @note required role:
-   *   - Security Token Owner
-   *
    * @deprecated
    */
   public removeAgent: ProcedureMethod<void, void>;
```

Comment:
Shouldn't we remove this procedure while we're at it?

Reply:
Not yet. The idea to keep the backward compatibility is to give users time to replace these methods with External Agents calls. 
Given that EA is not completely finished, we should keep them for a while. To make it easier to users to update, they should be available at the same time so they can refactor their code.
We will be removing these methods later.